unlock-app/src/components/interface/verifiers/VerifiersList.tsx

```diff
@@ -61,12 +88,18 @@ export const VerifiersList: React.FC<VerifiersListProsps> = ({
   const onConfirmDeleteVerifier = async () => {
     setLoading(true)
     const isValid = ethers.utils.isAddress(selectedVerifier)
+
     try {
       if (isValid) {
-        const addVerifierUrl = `/verifier/${network}/${lockAddress}/${selectedVerifier}`
+        const addVerifierUrl = getEndpoint(
```

Comment:
I wonder if this logic (the whole request) should be inside of the LocksmithService? wdyt @searchableguy

Reply:
Eventually, yes. For now, I will move this to storage service. 

I plan to migrate all methods from storage service to locksmith service and directly pass the instance of locksmith service from the library. That way, everyone can take advantage of it.